Notepad.Extensions.Logging/NotepadLogger.cs

```diff
@@ -12,14 +12,18 @@ namespace Notepad.Extensions.Logging
 {
     class NotepadLogger : ILogger
     {
-        public NotepadLogger(ObjectPool<StringBuilder> stringBuilderPool, string categoryName)
+        public NotepadLogger(ObjectPool<StringBuilder> stringBuilderPool, string categoryName, string windowName)
         {
             this.stringBuilderPool = stringBuilderPool;
             this.categoryName = categoryName;
+            this.windowName = windowName ?? throw new ArgumentNullException(nameof(windowName));
+            this.changedWindowName = $"*{windowName}";
```

Comment:
I'd rather just skip the `*` when searching (e.g. https://github.com/yaakov-h/Notepad.Extensions.Logging/blob/master/Notepad.Extensions.Logging/WindowFinder.cs#L45-L49) than keep track of it separately.

Reply:
Yes, since you added another search method it can be removed.